cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,57 @@
 /// <reference types='cypress' />
+import { generateWorker } from '../support/generate';
 
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('should check pagination', () => {
+    cy.get('.-pageInfo').should('be.visible').and('exist');
+    cy.get('[aria-label="rows per page"]').should('be.visible').and('exist');
+  });
+
+  it('should add new user', () => {
+    const { email, firstName, lastName, age, salary, department } =
+   generateWorker();
+    cy.get('#addNewRecordButton').click();
+    cy.get('#firstName').type(firstName);
+    cy.get('#lastName').type(lastName);
+    cy.get('#userEmail').type(email);
+    cy.get('#age').type(age.toString());
+    cy.get('#salary').type(salary.toString());
+    cy.get('#department').type(department);
+    cy.get('#submit').click();
+    cy.get('.rt-table').should('contain', firstName, lastName, email, age,
+      salary, department);
+  });
```

Comment:
This test case is good, but it would be even better if you added some assertions to check if the form fields have been filled correctly before submitting the form. This way, you can ensure that the data is correctly entered.